cmd/gpu_fakedev/gpu_fakedev.go

```diff
@@ -77,6 +82,11 @@ type genOptions struct {
 	devs  int
 }
 
+func min(val1 int, val2 int) int {
+	// TODO replace this with template version of Min(), once Golang standard library gets one
+	return int(math.Min(float64(val1), float64(val2)))
+}
```

Comment:
Latest Go release added generic `min()` & `max()` functions:
* https://go.dev/doc/go1.21#language
* https://go.dev/ref/spec#Min_and_max

I.e. this function can be removed when Intel k8s devices project upgrades to go v1.21.

Reply:
my thinking is to follow kubernetes, likely after our 0.28 release